gradle.properties

```diff
@@ -10,3 +10,5 @@ POM_LICENCE_NAME=The Apache Software License, Version 2.0
 POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_DEVELOPER_ID=pedrovgs
 POM_DEVELOPER_NAME=Pedro Vicente Gomez Sanchez
+
+android.useAndroidX=true
```

Comment:
This was required when running tests, there is probable other ways of fixing it.  Didn't want to add more to this PR